modules/main/src/main/env/service/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
@wasabi-honeybadger  regarding looks to replicate wasabi.sh, it is meant to be similar with the exception of 
```code
while ! nc -w 1 -z wasabi-cassandra 9042; do sleep 0.1; done
while ! nc -w 1 -z wasabi-mysql 3306; do sleep 0.1; done
```
which checks for the database to be up before running the java code.  The reason to do that is java code sometimes runs a lot faster than the time it requires to bring up a database.  So when the database is not ready, our java webapp should not start.  It is just a safety precaution.